Makefile

```diff
@@ -0,0 +1,49 @@
+pip:
+	pip install -e .
+
+pip-development: pip
```

Comment:
nit: if you view `pip` as a verb/idoim for "install pip dependencies for", these commands make sense, but I'm not sure I fully agree with that. Perhaps `setup-development`, `setup-docs`, etc. would be more clear?

Reply:
👍